### ISSUE TITLE
"last" panics when when input parameter is larger than the row of the output table

### ISSUE THREAD

**Issue by Nalleyer** [nushell/nushell#718]
# How to reproduce
```
> mkdir empty (ok)
> cd empty (ok)
> ls | first 1 (ok)
> ls | last 1 (crash)
```
# output
```
thread 'main' panicked at 'slice index starts at 18446744073709551615 but ends at 0', src\libcore\slice\mod.rs:2589:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```
#environment
nu 0.3.0 windows binary, downloaded from github releases.
windows 10, Intel CPU 7700
(sorry for my poor English)

**Comment by jonathandturner** [comment]
Thanks for reporting this! We'll take a look

**Comment by jonathandturner** [comment]
Fixed by #737
